popup.js

```diff
@@ -0,0 +1,120 @@
+/* Footer */
+const appFooter = `
+
+<div id="partners">
+<h1>Partners</h1>
+<hr />
+<ul>
+    <li>
+        <a href="https://github.com" target="_blank" rel="noopener noreferrer" tabindex="20"><img
+                src="./img/github-white.png" alt="github-icon" /></a>
+    </li>
+    <li>
+        <a href="https://www.linkedin.com" target="_blank" rel="noopener noreferrer" tabindex="21"><img
+                src="./img/linkedin-white.png" alt="linkedin-icon" /></a>
+    </li>
+    <li>
+        <a href="https://www.angellist.com/" target="_blank" tabindex="22"><img
+                src="./img/angellist-white.png" alt="angellist-icon" /></a>
+    </li>
+    <li>
+        <a href="https://twitter.com/" target="_blank" tabindex="23"><img src="./img/twitter-white.png"
+                alt="twitter-icon" /></a>
+    </li>
+    <li>
+        <a href="https://medium.com/" target="_blank" tabindex="24"><img src="./img/medium-white.png"
+                alt="medium-icon" /></a>
+    </li>
+</ul>
+</div>
+<div id="infor">
+<div id="infor-mobile">
+<div id="footer-info">
+    <img src="./img/logo2.png" alt="AISRC Logo" />
+    <div id="footer-info-text">
+        <h1>AISRC</h1>
+        <p>Annual International Science Research Conference</p>
+    </div>
+</div>
+    <p>
+        © AISRC 2023<br />
+        Right Reserved<br />
+        Since 1997 <br />
+        Contact: aisrc@question.org
+    </p>
+</div>
+</div>
```

Comment:
- [ ] I would implore you to use css flex to justify the content of this footer so that it would align as it is aligned in the Behance design.
- [ ] Also, the Home page background color is white but yours is dark. That of About page is dark and you have it as that, that is fine, kindly fix the Home page background color.
## Behance design
![image](https://user-images.githubusercontent.com/86200356/232955819-b675ae7c-419b-45ae-9d68-165377013ed0.png)
## Your Design
![image](https://user-images.githubusercontent.com/86200356/232956002-f942f1f4-74f7-4126-9ee4-fb600b55b285.png)

- [ ] Kindly take note that the mobile background color is white. Kindly fix it.
## Your design
![image](https://user-images.githubusercontent.com/86200356/232956438-43163521-8682-4569-8b60-136c63a95781.png)
## Behance design
![image](https://user-images.githubusercontent.com/86200356/232956513-1ff6b2f1-34d5-48f8-a3e2-c75dbe9326cf.png)

Reply:
The layout should have space between and around it. Do you understand?

Reply:
You are welcome.